trust/README.md

```diff
@@ -1,198 +1,437 @@
+   03e1855d4f31: Pull complete
+   a3ed95caeb02: Pull complete
+   ea9cb3d7d346: Pull complete
+   e47839e262bb: Pull complete
+   f568a56c1fd0: Pull complete
+   cc98c1dfbf81: Pull complete
+   98a99d2efdc4: Pull complete
+   0b304232c8e6: Pull complete
+   d65a44f4573e: Pull complete
+   Digest: sha256:10d0179f08a4fb0c785142ca73367921f46a93c2ee7c84831ae3543522156a6c
+   Status: Downloaded newer image for mariadb:10.1.10
+   <SNIP>
+   Creating notary_mysql_1
+   Creating notary_signer_1
+   Creating notary_server_1
+   ```
+
+5.  Add `127.0.0.1 notary-server` to `/etc/hosts` (or if using `docker-machine`, add `$(docker-machine ip) notary-server)`.
+
+6. Copy the config and certificate required to talk to your local Notary server.
+
+   ```
+   $ mkdir -p ~/.notary && cp cmd/notary/config.json cmd/notary/root-ca.crt ~/.notary
+   ```
+
+7.  Initialize a new trusted collection on your local server
+
+   ```
+   $ notary init example.com/scripts
+
+   No root keys found. Generating a new root key...
+You are about to create a new root signing key passphrase. This passphrase
+will be used to protect the most sensitive key in your signing system. Please
+choose a long, complex passphrase and be careful to keep the password and the
+key file itself secure and backed up. It is highly recommended that you use a
+password manager to generate the passphrase and keep it safe. There will be no
+way to recover this key. You can find the key in your config directory.
+Enter passphrase for new root key with ID 6f3b2d0:
+Repeat passphrase for new root key with ID 6f3b2d0:
+Enter passphrase for new targets key with ID 4bc290d (example.com/scripts):
+Repeat passphrase for new targets key with ID 4bc290d (example.com/scripts):
+   ```
+
+   You will be prompted for passphrases for the root and repository keys. This is for the same reasons as it was when you first pushed a repo with Docker Content Trust enabled.
+
+8.  Add content to your trusted collection by running a sequence of `notary add example.com/scripts <NAME> <FILE>`, `notary publish example.com/scripts`, and `notary list example.com/scripts`. This is a the sequence, in order:
+    - Stages adding a target with `notary add`
+    - Attempts to publish this target to the server with `notary publish`
+    - Fetches and displays all trust data for example.com (verifying output as it is downloaded)
 
-It is strongly encouraged to use official images whenever possible, as they are hand-curated and also signed!
+> **NOTE TO LAB OWNER. On my second run through the lab I get "fatal: could not rotate trust to a new trusted root: failed to validate data with current trusted certificates" when trying `notary publish example.com/scripts`, and `notary list example.com/scripts`. This could be a mistake on my behalf, or the fact it's a second run through, as I'm not familiar with Notary, but it would be good if you could add a bit of detail here in case I'm missing something. Cheers.**
```

Comment:
This is likely because of pre-existing notary data in its database volume from your first run through.  I can make a note about this.